packages/tre/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://laurikari.net/tre/
+TERMUX_PKG_DESCRIPTION="Approximate regex matching library and agrep CLI tool"
+TERMUX_PKG_LICENSE="BSD-2-Clause"
```

Comment:
That license identifier is malformed.
https://github.com/termux/termux-packages/blob/e7c982d40e86a28fc10967b4af9a9782b8bb7faa/scripts/lint-packages.sh#L18-L37

```suggestion
TERMUX_PKG_LICENSE="BSD 2-Clause"
```